Resources/doc/index.md

```diff
@@ -134,9 +134,13 @@ snc_redis:
             options:
                 replication: sentinel
                 service: mymaster
+                parameters:
+                    database: 1
+                    password: pass
 ```
 
 The `service` is the name of the set of Redis instances.
+The `parameters` is optional parameters that added to master/slave connections and ignores by sentinel connections.
```

Comment:
Can you please change this to:

The optional `parameters` option can be used to set parameters like the database number and password for the master/slave connections, they don't apply for the connection to sentinal.

(if it sounds correct to you, I don't have sentinal experience)